.vale/styles/foreman-documentation/Capitalization.yml

```diff
@@ -0,0 +1,31 @@
+# Suggest proper capitalization
+---
+extends: substitution
```

Comment:
Actually, extending the `substitution` check might not be the best solution.
There's a check for capitalization specifically: https://vale.sh/docs/checks/capitalization

It has exceptions :grin:

Reply:
I don't know. The scoping might be problematic.

Just a thought.